pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,23 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+
+- List the current settings:
+
+`efibootmgr -v for Filepaths`
+
+- Create a new boot option:
+
+`sudo efibootmgr -c -d {{esp_blockdevice}} -l {{\path\to\file.efi}} -L "{{Label}}"`
+
+- Add UEFI Shell v2 as a boot option:
```

Comment:
This is the same as above. We can just make the concrete examples part of that itself. Let's remove this one.

Reply:
Can you explain this more clearly? I don't quite understand you.

Reply:
I think what @agnivade means here is that this example is identical to the previous one in functionality, but unlike the example that appears directly before this one - this example contains concrete values instead of using the `{{token_syntax}}`. Hence, we can instead edit the previous example to use concrete values inside the `{{token_syntax}}` - and remove this example.

Reply:
That's exactly it.

Reply:
Not sure what's happened here, but this example doesn't look identical to the one before it anymore to me. I'mconfused?

/cc @agnivade

Reply:
This commit changed it https://github.com/tldr-pages/tldr/pull/3788/commits/f27372da2f82749602aedd8fed59cb563c80544a